NotesApplication/NotesApplication/Models/AppDbContext.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore;
+using NotesApplication.Models.Enums;
+
+namespace NotesApplication.Models
+{
+    public class AppDbContext : DbContext
+    {
+        public AppDbContext(DbContextOptions<AppDbContext> options) : base(options)
+        {
+
+        }
+        public DbSet<User> Users { get; set; }
```

Comment:
Bitno je i kako organizujes polja, konstruktore, propertije... Kao i to da uvek ide prazan red izmedju grupacije od istih, konkretno ovde treba prazan red izmedju propertija i konstruktora.